subworkflows/nf-core/fastq_align_chromap/main.nf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Map reads, sort, index BAM file and run samtools stats, flagstat and idxstats
+ */
+
+include { CHROMAP_CHROMAP         } from '../../../modules/nf-core/chromap/chromap/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_CHROMAP {
+    take:
+    reads // channel: [ val(meta), [ reads ] ]
+    index //    path: /path/to/index
+    fasta //    path: /path/to/fasta
+
+    main:
+    ch_versions = Channel.empty()
+
+    //
+    // Map reads with CHROMAP
+    //
+    CHROMAP_CHROMAP(reads, fasta, index, [], [], [], [])
```

Comment:
You need to add support (even if optional) for all the extra input files [^1]

[^1]: https://nf-co.re/docs/contributing/subworkflows#inputoutput-options point 1

Reply:
Addressed here 10dd532c63090e1f34ca2d67bf750dc25a145c60